### PR TITLE
LPS-204927

### PR DESCRIPTION
**Pull request by ealonso**
Hey @bakayattila,

I changed a little bit your proposal, doing it more conservative.

Instead of creating preferences only for view mode, doing it only if we are not in a preview mode, since we have other modes like edit and index, and maybe with this change we are going to break something.

Also, I created a new constant and methods for preview mode in fragment context.

Please let me know what do you think.

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by bakayattila** [comment]
Hi @ealonso,
There were so many use cases for protletPreferences generated with the wrong value. That I wanted to create it in a way where the view would be the exception, but I'm okay with your changes. I'm only not sure about one part.

File: modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/RenderFragmentEntryDisplayContext.java [hunk 1]
```diff
@@ -93,6 +94,8 @@ public DefaultFragmentRendererContext getDefaultFragmentRendererContext()
 		DefaultFragmentRendererContext defaultFragmentRendererContext =
 			new DefaultFragmentRendererContext(fragmentEntryLink);
 
+		defaultFragmentRendererContext.setMode(
+			FragmentEntryLinkConstants.PREVIEW);
```
Comment (bakayattila) [inline modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/RenderFragmentEntryDisplayContext.java:98]: Based on the commit message this should be conditional but I only see it set on the DefaultFragmentRendererContext without any condition. I'm only confused about this bit in the modifications.

Comment (ealonso) [reply modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/RenderFragmentEntryDisplayContext.java:98]: This code is only called when we are in the fragment entry edition, and only for the preview of the fragment entry. The pink box, for other fragment rendering we are using other logic.

Cache_-_Liferay

Comment (bakayattila) [reply modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/RenderFragmentEntryDisplayContext.java:98]: Thanks for the clarification!

**Comment by ealonso** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**